.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -4,12 +4,32 @@ labels: [Bug]
 body:
 - type: markdown
   attributes:
-    value: "# A330-900 Bug Report Form"
+    value: "# Headwind Simulations Bug Report Form"
+- type: dropdown
+  id: sim
+  attributes:
+    label: Simulator Version
+    description: Which simulator are you using?
+    options:
+    - 2020
+    - 2024
+  validations:
+    required: true
+- type: dropdown
+  id: aircraft
+  attributes:
+    label: Aircraft
+    description: What aircraft are you using?
+    options:
+    - A330-900x
+    - SU95x
+  validations:
+    required: true
 - type: dropdown
   id: version
   attributes:
     label: Aircraft Version
-    description: What version of the A330-900 are you using?
+    description: What version of the aircraft are you using?
     options:
     - Stable
     - Development
```

Comment:
This should be renamed to "Pre-Release" as we not providubg development version anymore.